sktime/forecasting/ets.py

```diff
@@ -180,9 +180,9 @@ class AutoETS(_StatsModelsAdapter):
 
     def __init__(
         self,
-        error="add",
+        error=None,
```

Comment:
Here I changed default values for error and damped_trend, so that the block of logic later can tell whether or not they have been passed in by the user.

Reply:
good idea and sth we often do, but see below on the "don't change init params" requirement